sqlglot/errors.py

```diff
@@ -47,3 +69,7 @@ def concat_errors(errors: t.Sequence[t.Any], maximum: int) -> str:
     if remaining > 0:
         msg.append(f"... and {remaining} more")
     return "\n\n".join(msg)
+
+
+def concat_error_props(errors: t.Sequence[ParseError], maximum: int) -> t.List[t.Dict[str, t.Any]]:
+    return [props for e in errors[:maximum] for props in e.error_props]
```

Comment:
We truncate the string so stacktraces don't explode.

But I don't think we have to truncate the list here, as its intended for programmatic use.

Reply:
Thanks for explaining, I was wondering about that. Fixed.